nucliadb/nucliadb/ingest/tests/servicer/test_list_members.py

```diff
@@ -34,3 +34,5 @@ async def test_list_members(grpc_servicer: IngestFixture):
 
     for member in response.members:
         assert member.type == Member.Type.IO
+        assert isinstance(member.load_score, float)
+        assert isinstance(member.shard_count, int)
```

Comment:
When it could be a different type?

Reply:
What do you mean ?

Reply:
It's testing that load_score and shard_count are of types float and int. I was wondering if they can be of a different type. As the functions to get the value always return the correct value and type